sdk/search/Azure.Search.Documents/README.md

```diff
@@ -289,15 +289,10 @@ AzureKeyCredential credential = new AzureKeyCredential(key);
 SearchIndexClient client = new SearchIndexClient(endpoint, credential);
 
 // Create the index using FieldBuilder.
-SearchIndex index = new SearchIndex("hotels")
-{
-    Fields = new FieldBuilder().Build(typeof(Hotel)),
-    Suggesters =
```

Comment:
This uses C#'s collection initializer pattern, which effectively calls `Add` on any `IEnumerable` that matches the input type(s). All these samples compile and are tested. Did you have problems compiling the sample with the requisite compiler version / SDK?

Reply:
Uh, think I was having a bad day yesterday! Yes, you're right - this works as expected. I've been migrating a project over from the deprecated Microsoft.Azure.Search and missed off the braces, doh!

Thank you for giving your time, and my apologies for a muggle mistake.